spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+  describe "GET start page" do
```

Comment:
It doesn't need to be nice, because this information is for developers only. Maybe it's better to use #index instead of start page?